radis/io/dbmanager.py

```diff
@@ -514,6 +516,27 @@ def get_nrows(self, local_file):
             raise ValueError(engine)
         return nrows
 
+    def get_columns(self, local_file):
```

Comment:
nice function. 

It would be nice to test it. There is already a test set-up that uses all engines, can you include the test for Pytables (pandas) and Vaex there : 

https://github.com/Supriya1702/radis/blob/fetch_hitran_update/radis/test/io/test_hdf5.py#L45

just assert() that the columns are what they are